src/bootslide.js

```diff
@@ -107,7 +131,27 @@ Bootslide.prototype.buildSections = function (menu, sections, back) {
 
     ul.append(li)
 
-    if (typeof target.target === 'function' || (typeof target.target === 'undefined' && self.defaultTarget)) {
+    function addAndContinue () {
+      li.addClass('bootslide-step')
+      // Keep digging
+      a.addClass('bootslide-scrollable')
+      a.prepend(self.next)
+      return self.buildSections(target, sections, true)
+    }
+
+    if (target.contentEndpoint) {
+      var newSection = addAndContinue()
+
+      function render (e) {
+        if (newSection.hasClass('bootslide-content-placeholder')) {
+          newSection.removeClass('bootslide-content-placeholder')
+          newSection.append(target.contentEndpoint())
+          newSection.css('width', '')
+        }
+      }
+
+      a.click(render)
```

Comment:
I feel like there was a reason it was a named function at some point, but that clearly is no longer the case.